lib/PuppeteerSharp/Helpers/RemoteObjectHelper.cs

```diff
@@ -8,8 +8,37 @@
 
 namespace PuppeteerSharp.Helpers
 {
-    internal class RemoteObjectHelper
+    internal static class RemoteObjectHelper
     {
+        internal static object FormatArgument(this IJSHandle jSHandle, ExecutionContext context)
```

Comment:
How about moving this to `PuppeteerHandleExtensions`?

Reply:
Done